Neos.ContentRepository/Classes/Domain/ValueObject/RootNodeIdentifiers.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Domain\ValueObject;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\ContentRepository\DimensionSpace\DimensionSpace\DimensionSpacePoint;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * Internal list of root node identifiers needed at various internal places.
+ *
+ * @Flow\Proxy(false)
+ */
+final class RootNodeIdentifiers
+{
```

Comment:
Can be removed along with NodeIdentifiers imho

Reply:
Nope, as the RootDimensionSpacePoint etc are needed in the codebase.

Reply:
Legacy or new? As for the new code base: not for much longer :)

Reply:
> Legacy or new?

new AFAIS – but we could still throw it out once the "new code base" is merged, no?